README.md

```diff
@@ -100,7 +100,12 @@ A glossary entry is structured like this:
         and the value may contain local links to other terms in this glossary
         (i.e., links starting with `#`)
         and/or links to outside sources.
+### Building a local test site
+Once you have finished creating your glossary entry it may be useful to build a local test site to check the syntax and formatting is correct. To do this you will need a working installation of Jekyll on your local machine (instructions for which can be found [here](https://jekyllrb.com/docs/installation/).
 
+With this, you can then use the command `make serve` this will create a local copy of the GitHub pages site that you can view in a web browser. The IP. address for this is listed in the command line output as `Server address: http://X.Y.Z:4000` however the default address is `http://localhost:4000`.
```

Comment:
```suggestion
With this, you can then use the command `make serve` this will create a local copy of the GitHub pages site that you can view in a web browser. The IP address for this is listed in the command line output as `Server address: http://X.Y.Z:4000` however the default address is `http://localhost:4000`.
```